_docs-sources/guides/build-it-yourself/1-overview.md

```diff
@@ -1,5 +1,61 @@
 ---
 sidebar_label: Overview
+pagination_label: Build It Yourself
 ---
 
+import Card from "/src/components/Card"
+import Grid from "/src/components/Grid"
+
 # Build Your Own Architecture
+
+Now that your foundational knowledge is in place and your workspace is configured, you’re ready to dive in and learn how to deploy production-grade infrastructure. If you’ve purchased a Gruntwork Reference Architecture, use [this guide](/docs/guides/reference-architecture/overview/overview) to get started. Otherwise, view our [courses](/docs/courses) and [guides](/docs/guides/welcome), or check out the [Service Catalog API reference](/docs/reference/services/intro) to learn what's available.
```

Comment:
Consider linking `foundational knowledge` and `workspace is configured` to the relevant intro sections (perhaps link to the stubs for now?)